x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -138,8 +138,13 @@ function getPathParams(pathname: string) {
   }
 }
 
+interface TimeUpdate {
+  min: string;
+  max: string;
+}
+
 // ACTION CREATORS
-export function updateTimePicker(time: string) {
+export function updateTimePicker(time: TimeUpdate) {
```

Comment:
This should be moved to `DatePicker.tsx` if that's the only place where it's consumed

Reply:
I think it makes sense to leave the action creator here where the reducer is, they share the action type constant value and I don't think we have many action creators in components anywhere else. It was only being used in one place before, too.

Reply:
Oh yes, we still use `TIMEPICKER_UPDATE` in the reducer, in urlParams. Forgot about that.